templates/SSDS-5.html.tmpl

```diff
@@ -598,7 +598,7 @@
 					</tbody>
                 </table>
             {{else}}
-                <p><em>N/A</em></p>
+                <p><em>There are no Bitbucket Pull Request (PR) merged yet.</em></p>
```

Comment:
Should not be plural the sentence "There are no Bitbucket Pull Requests (PRs) merged yet". 
Or something similar "No Bitbucket pull requests (PRs) have been merged yet."

Reply:
Updated to: "There is no Bitbucket Pull Request (PR) merged yet"